skops/_min_dependencies.py

```diff
@@ -27,6 +27,7 @@
     "sphinx-issues": ("1.2.0", "docs", None),
     "matplotlib": ("3.3", "docs", None),
     "pandas": ("1", "docs", None),
+    "gradio": ("3.3", "docs", None),
```

Comment:
If we do a user guide instead of an example file, we don't need this dependency.

Reply:
The tests were complaining that's why I had to add it back. I can remove again.

Reply:
Do the tests actually run the code inside the `rst` files?  I don't remember.